dev/io.openliberty.security.jakartasec.3.0.internal/resources/io/openliberty/security/jakartasec/internal/resources/JakartaSecurity30Messages.nlsprops

```diff
@@ -42,3 +42,7 @@ CREDENTIAL_VALIDATION_ERROR.useraction=For more information, see the error in th
 JAKARTASEC_WARNING_MISSING_SUBJECT_CLAIMS=CWWKS2505W: The claims JSON object in the OpenIdContext bean cannot add the subject value for client {0} because subject value is missing from the OpenIdClaims object. 
 JAKARTASEC_WARNING_MISSING_SUBJECT_CLAIMS.explanation=The OpenIdClaims object on the OpenIdContext bean is missing the required subject value. Therefore, the subject value of the claims JSON object cannot be set.  
 JAKARTASEC_WARNING_MISSING_SUBJECT_CLAIMS.useraction=Review the logs for earlier errors related to OpenID authentication.
+
+JAKARTASEC_CLAIMS_PROCESSING_ERROR=CWWKS2506E: The OpenID Connect client encountered an error while getting the {0} Claims key. The value is {1} and the expected type is {2}. The error is {3}.
```

Comment:
Can the first sentence of the error be made clearer? For example, instead of `The OpenID Connect client encountered an error while getting the {0} Claims key.` maybe `The OpenID Connect client {0} Claims key value is incorrect.`

I'll add the ID reviewed label to not hold up delivery while many are OOO.

Reply:
Thanks, wIll make the change